src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Lottery {
+    public static Colors getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```

method should return Ball